app/controllers/admin/users_controller.rb

```diff
@@ -54,24 +54,30 @@ def includes
   end
 
   def show_whitelist_attributes
-    %w[email name username two_factor_enabled? groups expires_at last_activity_at]
+    %w[email name username two_factor_enabled? groups roles expires_at last_activity_at]
   end
 
   def whitelist_attributes
-    %w[email username name two_factor_enabled? groups expired?]
+    %w[email username name two_factor_enabled? groups roles expired?]
   end
 
   def model
     User
   end
 
   # Never trust parameters from the scary internet, only allow the white list through.
-  def model_params
+  def model_params # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
     p = params.require(:user).permit(
       :email, :username, :email, :name, :expires_at,
       :password, :last_activity_at, group_ids: []
     )
     p[:group_ids] ||= []
+    # A Manager cannot add a user to an operator or admin group
```

Comment:
Should this comment be moved inside the condition?
Not 100% sure how the coding/commenting style if defined but the pluck action is more related to the comment than the manager condition itself.

Reply:
Good catch, will tidy this